SpikesBot.js

```diff
@@ -192,8 +190,13 @@ function searchSocialMedia(searchItem, teamNumber) {
 
 client.on('messageCreate', msg => {
 
-    //check that the message sent by a user
-    if (!msg.author.bot) {
+    const everyoneRole = msg.guild.roles.everyone;
+
+    //get the permissions of @everyone role in the channel
+    const permissions = msg.channel.permissionsFor(everyoneRole);
```

Comment:
check if the bot have permission and not everyone since the bot could be added for private channels

Reply:
discord is stupid this will not work